From 68062f9262cebd4d4fa14ad05d3a046bb0965e9a Mon Sep 17 00:00:00 2001
From: Timur Birsh <taem@linukz.org>
Date: Thu, 16 May 2013 13:44:32 +0600
Subject: [PATCH 2/3] Enable CFLAGS, CPPFLAGS and LDFLAGS for changing

For example, when building Debian packages, there is need to change compilation
flags due to hardening options.
---
 Makefile |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index ee23d90..c296209 100644
--- a/Makefile
+++ b/Makefile
@@ -26,8 +26,10 @@ BASE_OBJS     = src/base64utils.o src/md5.o src/dyndns.o src/errorcode.o src/get
 		src/http_client.o src/ip.o src/main.o src/os_unix.o src/os_windows.o   \
 		src/os.o src/os_psos.o src/tcp.o src/inadyn_cmd.o
 OBJS	      = $(BASE_OBJS) $(CFG_OBJ) $(EXTRA_OBJS)
-CFLAGS        = -Iinclude -DVERSION_STRING=\"$(VERSION)\" $(CFG_INC) $(EXTRA_CFLAGS)
-CFLAGS       += -O2 -W -Wall
+CFLAGS       ?= -g -O2
+CFLAGS       := -W -Wall -Iinclude -DVERSION_STRING=\"$(VERSION)\" $(CFG_INC) $(CFLAGS) $(EXTRA_CFLAGS)
+CPPFLAGS     ?=
+LDFLAGS      ?=
 LDLIBS       += -lresolv $(EXTRA_LIBS)
 DISTFILES     = README COPYING LICENSE
 
-- 
1.7.10.4

